lib/galaxy/util/__init__.py

```diff
@@ -2029,8 +2029,5 @@ def to_content_disposition(target: str) -> str:
 def validate_doi(doi: str) -> bool:
     if len(doi) > DOI_MAX_LENGTH:
         return False
-    prefix = "https://doi.org/|doi.org/|doi:"
-    doi_prefix = r"10\.\d+"
-    doi_suffix = r"\S+"
-    doi_re = re.compile(f"^{prefix}{doi_prefix}/{doi_suffix}$")
-    return bool(doi_re.match(doi))
+    doi_re = re.compile(r"(10\.(\d)+\/(\S+))")
```

Comment:
FYI DOIs are defined in https://www.doi.org/doi-handbook/DOI_Handbook_Final.pdf , in particular section 3.2 , which allows directory indicators different from "10" and prefixes with multiple full stops.
On the other hand, in a [small tool](https://github.com/TGAC/rfparser/) that I wrote to process DOIs for publications from my Institute, where I use a more relaxed RE `r"[\d.]+/.+"`, all 1500 DOIs still start with "10" and have only one full stop in the prefix. :man_shrugging:

Reply:
I think `10.` for a doi prefix is another good line to draw, this seems reliable for scientific identifiers for now